package.json

```
@@ -18,9 +18,19 @@
     "postbuild": "npm run types && npm run bundlesize",
     "prepublishOnly": "npm run build"
   },
-  "main": "dist/react-tooltip.cjs.min.js",
-  "module": "dist/react-tooltip.esm.min.js",
+  "main": "dist/react-tooltip.cjs",
+  "module": "dist/react-tooltip.mjs",
   "types": "dist/react-tooltip.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/react-tooltip.cjs",
```

```suggestion
      "types": "./dist/react-tooltip.d.ts",
      "require": "./dist/react-tooltip.cjs",
```

It is quite useful to add the `types` key to the exports maps as well. For reference, here is the TS documentation https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#packagejson-exports-imports-and-self-referencing.

The key needs to appear before require/import, as above.

---

Nice they finally supported exports map. Last time I checked the thread to support it, they still weren't sure how to do it. Thanks for the heads-up